test/suite/test_import10.py

```diff
@@ -87,7 +87,7 @@ def test_import_with_open_backup_cursor(self):
         cursor.close()
 
         all_files = self.take_full_backup(self.dir, bkup_c)
-        self.assertTrue(self.uri + "wt" not in all_files)
+        self.assertTrue(self.uri + ".wt" not in all_files)
```

Comment:
Realised that there was a small bug inside import10 test. let me know if I should be creating a ticket instead of just fixing it here.